src/Processors/Transforms/ColumnGathererTransform.h

```diff
@@ -191,7 +195,7 @@ void ColumnGathererStream::gather(Column & column_res)
         }
 
         source.pos += len;
-    }
```

Comment:
It's worth to change `allocatedBytes` to `bytes` like it was suggested in https://github.com/ClickHouse/ClickHouse/pull/59340#discussion_r1470354144. It appeared to be more important as it seemed. And also change this in https://github.com/ClickHouse/ClickHouse/blob/5a024dfc0936e062770d0cfaad0805b57c1fba17/src/Processors/Merges/Algorithms/MergedData.h#L125.

Reply:
Done.